sway-lib-std/src/bytes.sw

```diff
@@ -435,6 +435,48 @@ impl Bytes {
     pub fn is_empty(self) -> bool {
         self.len == 0
     }
+
+    /// Returns the SHA-2-256 hash of the elements.
+    ///
+    /// ### Examples
+    ///
+    /// ```sway
+    /// use std:bytes::Bytes;
+    ///
+    /// let bytes = Bytes::new();
+    /// bytes.push(1);
+    /// bytes.push(2);
+    /// bytes.push(3);
+    /// let sha256_hash = bytes.sha256();
+    /// ```
+    pub fn sha256(self) -> b256 {
+        let mut result_buffer: b256 = b256::min();
```

Comment:
```suggestion
        let mut result_buffer = b256::min();
```

Unless the compiler requires it we should avoid explicit annotation for types that can be inferred when reading the code